src/ui/MessageInput/index.jsx

```diff
@@ -523,7 +518,13 @@ const MessageInput = React.forwardRef((props, ref) => {
                   className="sendbird-message-input--attach-input"
                   type="file"
                   ref={fileInputRef}
-                  onChange={handleUploadFile(onFileUpload)}
+                  // It will affect to <Channel /> and <Thread />
+                  onChange={(event) => {
+                    onFileUpload([...event.currentTarget.files]);
```

Comment:
Do we need to copy the original array of files and make a new files array here? If not, this would just work.

```suggestion
                    onFileUpload(event.currentTarget.files);
```

Reply:
Oh, this is necessary I think.
The `event.currentTarget` object becomes empty soon because I reset the value right below.
```
event.target.value = '';
```